libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/api/MapboxRouteArrowView.kt

```diff
@@ -33,12 +35,38 @@ import com.mapbox.navigation.utils.internal.logE
  * @param options the options used for determining the rendering appearance and/or behavior.
  */
 @UiThread
-class MapboxRouteArrowView(private val options: RouteArrowOptions) {
+class MapboxRouteArrowView(
+    private var options: RouteArrowOptions,
+    private var rebuildLayersOnFirstRender: Boolean = true
```

Comment:
Let's not open up this option as a constructor parameter and just execute the logic on each instance creation.

Reply:
The route arrow layers are dependent on the route line layers being in place first.  Creating the arrow layers on instantiation isn't an option because it's unlikely the route line layers will have been initialized. This is why the layers are checked and potentially initialized on render calls.

Reply:
Yes, sorry for not being specific - I mean let's always initialize the layers on first render call. Let's not open it up as an option _if_ they should be initialized on first render call, let's just always do it.

Reply:
Let's adjust unit tests instead of exposing the constructor.